test/functional.js

```diff
@@ -603,5 +603,84 @@ module.exports = {
                 done();
             });
         });
+
+        it('Vue.js is compiled correctly', (done) => {
+            const cwd = process.cwd();
+            after(() => {
+                process.chdir(cwd);
+            });
+
+            const appDir = testSetup.createTestAppDir();
+            /*
+             * Most of the time, we don't explicitly use chdir
+             * in order to set the cwd() to the test directory.
+             * That's because, in theory, you should be able to
+             * run Encore from other directories, give you set
+             * the context. However, in this case, the vue-loader
+             * users load-postcss-config to load the postcss config,
+             * but it uses process.cwd() to find it, instead of the
```

Comment:
is the context an Encore thing or a webpack one ? If it is a webpack one, it would be worth discussing this upstream